src/slideseq/plot/plot_downsampling.py

```diff
@@ -131,11 +133,25 @@ def model_least_squares(params, *, r, data):
     ax.set_ylabel("Transcripts per matched barcode")
     ax.set_title("Average transcripts for matched barcodes")
 
-    # r went upto 1.0 for actual data, but x_values for model go up to 3.0
-    ax.set_xlim(0.0, 3.1)
+    ax.set_xlim(
+        0.0, 3.1
+    )  # r went upto 1.0 for actual data, but x_values for model go up to 3.0
 
     ax.legend()
 
+    # calculate 2x and 10x depth for the 100% model with the ratios model(2) / model(1) and model(10) / model(1)
+    r_2 = (model(2.0, params)/model(1.0, params)) # model(2) / model(1)
+    r_10 = (model(10.0, params)/model(1.0, params)) # model(10) / model(1)
```

Comment:
Cleaning this up a little bit: removing unneeded parentheses and adding spaces. The comments don't really add more info to the code so I cut them.
```suggestion
    r_2 = model(2.0, params) / model(1.0, params)
    r_10 = model(10.0, params) / model(1.0, params)
```